src/lib/models/keyframe.js

```diff
@@ -41,12 +47,12 @@ module.exports = class Keyframe {
   }
 
   componentVars(componentName) {
-    return _.get(this.services, `componentName}.variables`, [])
+    return _.get(this.services, [componentName, 'variables'], [])
   }
 
   services() {
     if (_.has(this, 'services')) return this.services
-    if (_.has(this, 'components')) return this.components
+    if (_.has(this, 'components')) return _.get(this, 'components')
```

Comment:
Why this change?

Reply:
The only explanation I can think of from this is that your editor might think that `this` might not be an object, and therefore something ike `this.property` would result in an uncaught exception, and Lodash's `_.get` can handle this edge case, but hard to say. What editor are you using? Can you show us a screenshot of the actual error?